Sources/Options.swift

```diff
@@ -0,0 +1,95 @@
+//
+// Options.swift
+//
+// Copyright 2017 Handsome LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+/**
+ Basic spring params.
+ */
+public struct SpringParams {
+    
+    /// The damping ratio from 0 to 1 for the spring animation as it approaches its quiescent state.
+    let dampingRatio: CGFloat
+    
+    /// The initial spring velocity. For smooth start to the animation, match this value to the view’s velocity.
+    let velocity: CGFloat
+}
```

Comment:
@okonor sorry, you're right. I just thought about another thing in that moment. 😦 I will resolve your comments today!